tfmigrate/multi_state_migrator_test.go

```diff
@@ -90,18 +113,19 @@ func TestAccMultiStateMigratorApply(t *testing.T) {
 	ctx := context.Background()
 
 	fromBackend := tfexec.GetTestAccBackendS3Config(t.Name() + "/fromDir")
+	fromWorkspace := "work1"
```

Comment:
I want to confirm this change doesn't break `default` workspace behavior.
Can we parameterize the `fromWorkspace` and `toWorkspace` variables with the table-driven test?

Reply:
Or just copy and paste to a new test case such as `TestAccMultiStateMigratorApplyWithWorkspace`.

Reply:
I have followed your suggestion to use table-driven test. Please let me know what you think about the new test and if any changes are required.

Reply:
Thanks. Great work 👍